tensorflow_datasets/testing/dataset_builder_testing_test.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test for checksum in tensorflow_datasets.testing.dataset_builder_testing."""
+
+from __future__ import absolute_import
```

Comment:
Please remove all the `from __future__ import *` as we are not providing python2 compatibility anymore

Reply:
Thanks @vijayphoenix for pointing this out I have done the same.